packages/nodes-base/nodes/EmailSend/EmailSend.node.json

```
@@ -2,7 +2,10 @@
 	"node": "n8n-nodes-base.emailSend",
 	"nodeVersion": "1.0",
 	"codexVersion": "1.0",
-	"categories": ["Communication", "Core Nodes"],
+	"categories": ["Communication", "HILT"],
```

```suggestion
	"categories": ["Communication", "HILT", "Core Nodes"],
```
We still need this to be in Core Nodes, no?